src/MovieSnapShot/MovieSnapShot.js

```diff
@@ -1,51 +1,79 @@
-import React from "react";
+import React, { Component } from "react";
 import Header from "../Header/Header";
 import Footer from "../Footer/Footer";
 import MovieCard from "../MovieCard/MovieCard";
 import "./MovieSnapShot.css";
 import PropTypes from "prop-types";
+import { getSingleMovieData } from "../apiCalls.js"
+import { singleMovieCleaner, trailerCleaner } from "../utility.js";
 
-    const MovieSnapShot = ({ singleMovie, trailer, onHomeClick, error}) => {
-    const trailerToPlay = `https://www.youtube.com/embed/${trailer.key}`;
```

Comment:
We may be able to set this to state when we do the fetch. The only video data used is for this variable.

Reply:
agree, we should do this refactor